docs/sources/tempo/configuration/_index.md

```diff
@@ -1460,6 +1556,48 @@ parquet_dedicated_columns: <list of columns>
       [scope: <string>]
 ```
 
+### Compaction config
+
+The compaction configuration block is used by the compactor, backend scheduler, and backend worker.
+Refer to the [Compactor](#compactor) section for the full list of compaction configuration options.
+
+```yaml
+# Optional. Duration to keep blocks. Default is 14 days (336h).
+[block_retention: <duration>]
+
+# Optional. Duration to keep blocks that have been compacted elsewhere. Default is 1h.
+[compacted_block_retention: <duration>]
+
+# Optional. Blocks in this time window will be compacted together. Default is 1h.
+[compaction_window: <duration>]
+
+# Optional. Maximum number of traces in a compacted block. Default is 6 million.
+# WARNING: Deprecated. Use max_block_bytes instead.
+[max_compaction_objects: <int>]
+
+# Optional. Maximum size of a compacted block in bytes. Default is 100 GB.
+[max_block_bytes: <int>]
+
+# Optional. Number of tenants to process in parallel during retention. Default is 10.
+[retention_concurrency: <int>]
+
+# Optional. The maximum amount of time to spend compacting a single tenant before moving to the next. Default is 5m.
+[max_time_per_tenant: <duration>]
+
+# Optional. The time between compaction cycles. Default is 30s.
+# Note: The default will be used if the value is set to 0.
+[compaction_cycle: <duration>]
+
+# Optional. Amount of data to buffer from input blocks. Default is 5 MiB.
+[v2_in_buffer_bytes: <int>]
+
+# Optional. Flush data to backend when buffer is this large. Default is 20 MB.
+[v2_out_buffer_bytes: <int>]
+
+# Optional. Number of traces to buffer in memory during compaction. Increasing may improve performance but will also increase memory usage. Default is 1000.
+[v2_prefetch_traces_count: <int>]
+```
+
 ### Filter policy config
```

Comment:
Same question here about config in the headings -- and for any other headings that discuss a block of headings.